lib/fluent/plugin/out_vmware_loginsight.rb

```diff
@@ -70,7 +70,16 @@ class ConnectionFailure < StandardError; end
       config_param :flatten_hashes, :bool, :default => true
       # Seperator to use for joining flattened keys
       config_param :flatten_hashes_separator, :string, :default => "_"
-
+      # Keys from log event whose values shorten
```

Comment:
Can you add an  example here to make it more readable for users. Also, ruby does follow the insertion order for literals, but not sure if it's guaranteed. May be just note that too in the comment.

Reply:
updated